README.md

```diff
@@ -166,6 +166,26 @@ For more information, see:
 [Metrics API design]: https://github.com/kubernetes/design-proposals-archive/blob/main/instrumentation/resource-metrics-api.md
 [Metrics Server design]: https://github.com/kubernetes/design-proposals-archive/blob/main/instrumentation/metrics-server.md
 
+This diagram shows how metrics-server handles a `kubectl top pods` request:
+```mermaid
+sequenceDiagram
+    participant Client
+    participant APIServer
+    participant MS as api.podMetrics
```

Comment:
Would be good to be able to recognize participants that are part of Metrics Server and which are not. I also don't know if mentioning internal implementation details like structures `storage.Storage` brings any value or would just confuse people that don't know them.

 Could maybe just simplify the diagram to have metrics server as one participant and maybe create a separate diagram for internal component architecture?

Reply:
Maybe we could squeeze in how metrics are collected from Kubelets

Reply:
Simplified.